src/Aquifer.API/Modules/Resources/LanguageResources/LanguageResourcesEndpoints.cs

```diff
@@ -123,24 +137,32 @@ public static async Task<Results<Ok<ResourceItemsByChapterResponse>, NotFound, B
                 Contents = g
                     .Select(item => item.Content)
                     .DistinctBy(item => item.ContentId)
+                    .OrderBy(item => item.ContentId)
```

Comment:
This isn't necessary for the bug fix but having a consistent ordering here made comparisons easier when testing.